recipes/ndarray_listener/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "ndarray_listener" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "b921471c875eb7e79f8c63c5eaeafa0223580ad63c251ea6a44b0af2458bd960" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest
+    - six
+  run:
+    - python
+    - numpy
+    - pytest
+    - nose
```

Comment:
So even though it is possible to supply something to run the tests at run time. We don't typically do this as these are optional dependencies that are not typically needed. This is mainly following Continuum's lead. So things like `numpy` and `scipy` will need `nose` installed if we wish to run their test suites as `nose` is not pulled in by default. The same strategy should be followed here and with the rest of the proposed recipes. Unless, of course, their goal is to be a plugin to a testing framework.